references/classification/utils.py

```diff
@@ -463,3 +465,13 @@ def _add_params(module, prefix=""):
         if len(params[key]) > 0:
             param_groups.append({"params": params[key], "weight_decay": params_weight_decay[key]})
     return param_groups
+
+def set_seed(seed: int):
+    """
+    Function for setting all the RNGs to the same seed
+    """
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
+    np.random.seed(seed)
+    random.seed(seed)
```

Comment:
I think we shouldn't offer this because PyTorch doesn't guarantee reproducibility for the random generator across versions. See [here](https://pytorch.org/docs/stable/notes/randomness.html).